cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -119,6 +126,33 @@ protected void invoke(CommandContext context) throws Exception {
         }
     }
 
+    private String promptHelidonVersion(ArchetypesData archetypesData, boolean showLatest) {
+        String defaultHelidonVersion = defaultHelidonVersion(archetypesData);
+        List<String> versions = Lists.map(archetypesData.versions(), Version::id);
+        if (showLatest) {
+            versions = Lists.map(archetypesData.latestMajorVersions(versions), MavenVersion::toString);
```

Comment:
Same comment as above, can we have a method in archetypesData return what we need here.